packages/x-components/tests/e2e/cucumber/mocked-responses.spec.ts

```diff
@@ -0,0 +1,558 @@
+import {
+  NextQueriesResponse,
+  RelatedTagsResponse,
+  SearchByIdResponse,
+  SearchResponse,
+  SuggestionsResponse,
+  TopRecommendationsResponse
+} from '@empathyco/x-adapter';
+import { Given } from 'cypress-cucumber-preprocessor/steps';
+import {
+  createHierarchicalFacetStub,
+  createNextQueryStub,
+  createNumberRangeFacetStub,
+  createPopularSearch,
+  createQuerySuggestion,
+  createRelatedTagStub,
+  createResultStub,
+  createSimpleFacetStub,
+  getFacetsStub,
+  getNextQueriesStub,
+  getPopularSearchesStub,
+  getQuerySuggestionsStub,
+  getRelatedTagsStub,
+  getResultsStub
+} from '../../../src/__stubs__';
+
+// ID Results
+Given('an ID results API', () => {
+  cy.intercept('https://api.empathy.co/searchById', req => {
```

Comment:
Now that the `intercept`are all in the same place we can extract the mocked API URL to a constant and make all the `intercept` use that instead.
Also, extract the API endpoints that are repeated. I'm thinking of something like:

```javascript
const mockedApiUrl = 'https://api.empathy.co'

// ID Results
const searchByIdEndpoint = `${mockedApiUrl}/searchById`;

Given('an ID results API', () => {
  cy.intercept(searchByIdEndpoint, req => {
[...]

// Partial Results
Given('a results API with partial results', () => {
  cy.intercept(`${mockedApiUrl}/search`, req => {
[...]
```